### PR TITLE
[BUD 769] Output Rendered PDF to S3

### PR DESCRIPTION
**Pull request by mboperator**
## Description 

This branch stores generated PDFs in an S3 bucket, and returns a link to said PDF at the end of a successful render.

## Changes
- Added S3 IAM configs
- Removed rollup, due to packaging issues with `aws-sdk`
- Added webpack
- Stringify response `body` to avoid `502` when making requests to endpoint

## Screenshots
screen shot 2016-11-28 at 4 21 34 pm

### CODE REVIEW

**Comment by pnispel** [comment]
https://github.com/mboperator/pdfreactor/pull/1/commits/8270756e7caff31f69ea0707f819e2cd121a8789 ??

**Comment by mboperator** [comment]
@pnispel It made the diff hard to grok. Should we commit our `yarn.lock`? A contributor on `redux-modules` gitignored it in that repo. I'm not sure what the best practice is.

**Comment by pnispel** [comment]
I'm pretty sure the lockfiles need to be committed to be useful. At least if its similar to npm's shrinkwrap lockfiles. http://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        test: /\.json$/,
+        loader: 'json',
+      },
+    ],
+  },
+  node: {
+    __dirname: true,
+  },
+  externals: {
+    'aws-sdk': 'aws-sdk',
```
Comment (thewillhuang) [inline webpack.config.js:26]: can just do
 ```js
externals: ['aws-sdk'],
```

**Review by thewillhuang — APPROVED** [review]

**Comment by dannyphillips** [comment]
yes, you're right @pnispel

**Pull request merged**